suite-common/suite-types/package.json

```diff
@@ -19,5 +19,10 @@
     "devDependencies": {
         "jest": "^26.6.3",
         "typescript": "4.7.4"
+    },
+    "nx": {
```

Comment:
This is not right solution. Please move that `messageTypes.ts` file from `@suite-common/suite-types` package to this package. This is how I meant it in first place because it makes sense to have everything together and you won't even need "hacks" like implicit dependencies.